### DISCUSSION TITLE
create a dense pointcloud from an existing sparse pycolmap.reconstruction object

### DISCUSSION THREAD

**Discussion by shubham-monarch** [colmap/colmap#2257] [General]
I was tinkering around with [PixSFM](https://github.com/cvg/pixel-perfect-sfm). I ran the [demo.ipynb](https://github.com/cvg/pixel-perfect-sfm/blob/main/demo.ipynb) notebook on my custom dataset to generate a `pycolmap.reconstruction` object. However, the pointcloud generated appears to be quite sparse. So, I was looking around ways to densify it. I came across colmap `stereo_fusion`. I was wondering if `pycolmap` allows us to densify an existing `pycolmap.Reconstruction` object?

**Comment by HernandoR** [comment]
it has been a year after, any update?
